kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/screenshots/screenshotmaker/ExternalScreenshotMaker.kt

```diff
@@ -23,4 +23,8 @@ class ExternalScreenshotMaker(
     override fun takeScreenshot(file: File) {
         device.takeScreenshot(file, scale, params.quality)
     }
+
+    override fun takeFullScreenshot(file: File) {
```

Comment:
@Areyana I find this solution not good enough. There are no full view screenshots here. We can make this method unimplemented, add full view implementation for this class(may be not possible), or add some comments/TODO here.